schemas/replit.toml.json

```diff
@@ -0,0 +1,218 @@
+                "python_beta",
+                "html_beta",
+                "testj_beta",
+                "scala",
+                "riddlejs",
+                "java_maven"
+            ]
+        },
+        "audio": {
+            "description": "Enable/disable system-wide audio in REPL",
+            "type": "boolean"
+        },
+        "packager": {
+            "description": "Universal Package Manager (UPM) configuration",
+            "type": "object",
+            "properties": {
+                "afterInstall": {
+                    "description": "Command to run on new package install",
+                    "type": "string"
+                },
+                "ignoredPaths": {
+                    "description": "List of ignored path",
+                    "type": "array",
+                    "items": {
+                      "type": "string",
+                      "minLength": 1  
+                    },
+                    "uniqueItems": true,
+                    "minItems": 1
+                },
+                "ignoredPackages": {
+                    "description": "List of ignored packages",
+                    "type": "array",
+                    "items": {
+                      "type": "string",
+                      "minLength": 1  
+                    },
+                    "uniqueItems": true,
+                    "minItems": 1
+                },
+                "language": {
+                    "description": "Language name",
+                    "type": "string",
+                    "enum": [
+                        "python-python3-poetry",
+                        "python-python2-poetry",
+                        "nodejs-npm",
+                        "nodejs-yarn",
+                        "ruby-bundler",
+                        "elisp-cask",
+                        "dart-pub",
+                        "java-maven",
+                        "rlang",
+                        "dotnet",
+                        "rust"
+                    ]
+                },
+                "features": {
+                    "description": "Universal Package Manager (UPM) features",
+                    "type": "object",
+                    "properties": {
+                        "packageSearch": {
+                            "description": "Enable/disable package search panel",
+                            "type": "boolean"
+                        },
+                        "guessImports": {
+                            "description": "Enable/disable guessing required packages",
+                            "type": "boolean"
+                        }
+                    },
+                    "additionalProperties": false
+                }
+            },
+            "additionalProperties": false
+        },
+        "languages": {
+            "description": "Language configuration",
+            "type": "object",
+            "patternProperties": {
+                ".": {
```

Comment:
This will match a single character, is this correct?

Reply:
Because regex are not anchored by default it means that language name is at least one any char. I don't know exactly valid language identifier format for Repl.it to write smth better.

Reply:
Ah I see, my bad. In this case we could probably skip a regex match by using `additionalProperties` instead, but that would allow empty keys as well. We could also leave it as-is, up to you.

Reply:
I prefer not to use `additionalProperties: true` until they are permitted in the tool documentation. 😄